sigmf-spec.md

```diff
@@ -106,6 +106,21 @@ file contains information that describes the dataset.
 3. The metadata file MUST have a `.meta` filename extension.
 4. The dataset file MUST have a `.data` filename extension.
 
+#### Archive Format
+
+The metadata and dataset files that comprise a SigMF recording may be combined
+into a file archive. Multiple SigMF recordings may be stored in a single
+archive.
+
+1. The archive MUST use the `tar` archive format, as specified by POSIX.1-2001.
+2. The archive file's filename extension MUST be `.sigmf`.
+3. Metadata and dataset files MUST NOT be in the top-level directory of the
+   archive.
+4. Each SigMF recording (comprised of a metadata file and a dataset file) MUST
+   be in separate directories.
+5. The metadata and dataset files within a directory must have the same
+   filenames as the directory itself.
```

Comment:
I don't like how point 3 is written negatively; it does not rule out that you could have two levels of directories (which is another complication), or using absolute paths (which would be bad).

I would suggest something to the effect of (I don't know what language POSIX uses to describe tar files):

3. The archive MUST contain the following files: for each contained recording with some name given here metasyntactically as `N`, files named `N` (a directory), `N/N.sigmf-meta`, and `N/N.sigmf-data`.

4. The archive MUST NOT contain any other files unless their pathnames begin with `N/N.` for some `N` which has `.sigmf-meta` and `.sigmf-data` files as described above.

(The item 4 is to allow for extensions without allowing for arbitrary junk. More minimally, we could say that for now no other files are allowed.)